src/main/webapp/i18n/de/tutorialGroupExport.json

```
@@ -0,0 +1,14 @@
+{
+    "artemisApp": {
+        "tutorialGroupExportDialog": {
+            "buttonHeader": "Tutoriumsgruppen exportieren",
+            "header": "Bitte wähle die zu exportierenden Felder aus",
+            "description": "Bitte mindestens ein Feld für den CSV-Export auswählen:",
+            "toolTip": "Dieser Dialog kann verwendet werden, um Tutoriumsgruppen in eine CSV-Datei zu exportieren. Du kannst auswählen, welche Informationen du in deine CSV-Datei aufnehmen möchtest.",
```

```suggestion
            "toolTip": "Dieser Dialog kann verwendet werden, um Tutoriumsgruppen in eine CSV-Datei oder JSON-Datei zu exportieren. Du kannst auswählen, welche Informationen du aufnehmen möchtest.",
```
I thought json is possible as well?

---

yes fixed it 👍